python/GUI/CMakeLists.txt

```diff
@@ -23,15 +23,26 @@ GR_PYTHON_INSTALL(
     FILES
     usrp_dab_rx.py
     usrp_dab_tx.py
-    user_frontend.py
+    ${GUI_PY}
```

Comment:
I'm under the impression that you forgot to move the `set(GUI_PY…` lines **above** this line, so that this works?